src/penn_chime/validators/validators.py

```diff
@@ -47,7 +47,7 @@ def __init__(self) -> None:
         pass
    
     def validate(self, value):
-        if 0 >= value or value >= 1:
+        if 0 > value or value > 1:
```

Comment:
I zero a valid market share?

Maybe `if not 0 < value <= 1.0:` python supports chained conditions like this I think.